test_util/launch_configs/aws-linux-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+---
+deployment_name: dcos-stack-that-was-provisioned-from-vpc
+deployment_type: linux_cluster
+provider: aws
+provider_info:
+    region: us-west-2
+    access_key_id: ALSAFLAJ4L20922KF20TL294
+    secret_access_key: ADWOFIAOFWJTPW3924JR
+key_helper: true
+linux_cluster:
```

Comment:
I think you might be mis-interpreting what this is specifying. This is going to *create* those instances in that count. There is no picking involved.

As I just commented, the deployment name is wrong in this example. This example is for creating a plain linux cluster of homogeneous instances

the field os_name is dcos/dcos/test_util/aws specific. We have a mapping from strings to region to AMI. For each of those OS-strings, we also know the default user and home dir. There is no reason for the user to provide information for hard-coded AMIs. No passwords are ever used anywhere (key only) so that is irrelevant.